python/lsst/meas/deblender/deblend.py

```diff
@@ -467,67 +470,67 @@ class MultibandDeblendConfig(pexConfig.Config):
-                                         doc=("Whether or not to fit multiple components for each peak"))
-    constraints = pexConfig.Field(dtype=str, default="S",
-                                  doc=("List of constraints to use for each object"))
+    constraints = pexConfig.Field(dtype=str, default="1+SM",
+                                  doc=("List of constraints to use for each object."
+                                       "Current options are all used by default:\n"
+                                       "S: symmetry\n"
+                                       "M: monotonicity\n"
+                                       "1: normalized SED to unity"
+                                       "+: non-negative morphology"))
+    symmetryThresh = pexConfig.Field(dtype=float, default=1.0,
+                                     doc=("Strictness of symmetry from"
+                                          "0 (no symmetry enforced) to"
+                                          "1 (perfect symmetry required)."
+                                          "If 'S' is not in `constraints`, this argument is ignored"))
     l0Thresh = pexConfig.Field(dtype=float, default=np.nan,
                                doc=("L0 threshold. NaN results in no L0 penalty."))
     l1Thresh = pexConfig.Field(dtype=float, default=np.nan,
                                doc=("L1 threshold. NaN results in no L1 penalty."))
-    useStrictMonotonicity = pexConfig.Field(dtype=bool, default=True,
-                                            doc=("Whether to use strict monotonicity as a constraint"))
-    useNearestMonotonic = pexConfig.Field(dtype=bool, default=True,
-                                          doc=("Whether to use the nearest neighboring pixel for the"
-                                               "monotonicity constraint or a weighted average of the"
-                                               "3 neighboring pixels closest to the peak for each pixel"))
-    smoothness = pexConfig.Field(dtype=float, default=1,
-                                 doc=("Threshold on the l1 norm for smoothness constraints"))
-    # Other Deblender
-    stepSlack = pexConfig.Field(dtype=float, default=0.9,
-                                doc=("Slack to use when updating the step size in each iteration of"
-                                     "the bSDMM minimization"))
-    usePsfConvolution = pexConfig.Field(dtype=bool, default=False, doc=("Peform PSF kernel matching?"))
-    updateMorphologyFirst = pexConfig.Field(dtype=bool, default=True,
-                                            doc=("Whether to update morphology or SED first"))
+    tvxThresh = pexConfig.Field(dtype=float, default=np.nan,
+                                doc=("TVx threshold. NaN results in no TVx penalty."))
+    tvyThresh = pexConfig.Field(dtype=float, default=np.nan,
+                                doc=("TVy threshold. NaN results in no TVy penalty."))
```

Comment:
Can we always be sure that the meaning of TV is clear to the user from context?

Reply:
I think so, but I clarified these variables slightly.